template/.github/workflows/ci.yml

```diff
@@ -20,5 +21,8 @@ jobs:
         pip install invoke poetry
         poetry install
 
-    - name: Run static analysis
-      run: poetry run invoke format
+    - name: Run linting and formatting
+      run: |
+        poetry run invoke lint
+        poetry run invoke format
```

Comment:
<!-- Issues highlight specific problems with the subject under review. These problems can be user-facing or behind the scenes. It is strongly recommended to pair this comment with a suggestion. If you are not sure if a problem exists or not, consider leaving a question.  -->
**issue:** This only applies automatic formatting

The `format` task currently only runs isort and black in their default mode which _updates_ the code to their standards. With the Continuous integration pipelines we just want them to _check_ the formatting and fail if the formatting is not correct.

I'd suggest adding an optional `--check` flag to the `invoke` task. That way you can continue to the use the task locally with just `invoke format` but when run in CI you would use `invoke format --check` to just perform checking. Both isort and black have a `--check` option to do this.‏

Reply:
Updated in [e667181](https://github.com/ackama/django-template/pull/44/commits/e66718141dd30193d4b151069df290eb97c497ed) and [e01e35f](https://github.com/ackama/django-template/pull/44/commits/e01e35ff2afc75fbd42546a2ac5bce92eebc3d67)